src/Makefile.leveldb.include

```diff
@@ -26,7 +26,7 @@ LEVELDB_CPPFLAGS_INT += -DLEVELDB_PLATFORM_POSIX
 endif
 
 leveldb_libleveldb_a_CPPFLAGS = $(AM_CPPFLAGS) $(LEVELDB_CPPFLAGS_INT) $(LEVELDB_CPPFLAGS)
-leveldb_libleveldb_a_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
+leveldb_libleveldb_a_CXXFLAGS = -fPIC $(AM_CXXFLAGS) $(PIE_FLAGS)
```

Comment:
makes sense

Reply:
It violates the 'don't touch what isn't broken'-principle, though - which makes me feel a little uneasy. I am not aware of any potential problems here, but then I didn't look very deeply into this.

Might there be any potential stack smashing protection logic or similar that is impacted by this?

Reply:
Could be a problem and I agree that we are going against the "if it isn't broken does not fix it" mantra. Does the `bitcoind` produced using this new make file template work normally?